src/Information/InformationModal.tsx

```diff
@@ -0,0 +1,71 @@
+import InformationItem from './InformationItem.tsx';
+import {
+  Accordion,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay
+} from '@chakra-ui/react';
+import React, { useState } from 'react';
+
+interface InformationModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+const InformationModal: React.FC<InformationModalProps> = ({
+  isOpen,
+  onClose
+}) => {
+  const [isAccordionOpen, setIsAccordionOpen] = useState<number | null>(null);
+
+  const handleAccordionClick = (index: number) => {
+    if (isAccordionOpen === index) {
+      setIsAccordionOpen(null);
+    } else {
+      setIsAccordionOpen(index);
+    }
+  };
```

Comment:
Why are we managing the open item? Looking at the chakra docs it should manage the open item on its own

Reply:
fixed?